server/src/services/app_config.service.ts

```diff
@@ -38,4 +38,28 @@ export class AppConfigService {
 
     return whitelistedConfigVars;
   }
+
+  async constructSSOConfigs() {
```

Comment:
Shall we move this to organizations services since its used only by organization module? and we can remove 
`SSO_GOOGLE_OAUTH2_CLIENT_ID`
`SSO_GIT_OAUTH2_CLIENT_ID`
`SSO_GIT_OAUTH2_HOST`
`DISABLE_SIGNUPS`
`SSO_DISABLE_SIGNUPS`
from public configs since its served by API